src/test/java/org/springframework/data/elasticsearch/core/ElasticsearchTransportTemplateCallbackTests.java

```diff
@@ -68,10 +62,24 @@
 	@Mock private ActionFuture<BulkResponse> bulkResponseActionFuture;
 	@Mock private BulkResponse bulkResponse;
 	@Mock private BulkItemResponse bulkItemResponse;
+	@Mock private GetRequestBuilder getRequestBuilder;
+	@Mock private ActionFuture<GetResponse> getResponseActionFuture;
+	@Mock private GetResponse getResponse;
+	@Mock private MultiGetRequestBuilder multiGetRequestBuilder;
+	@Mock private ActionFuture<MultiGetResponse> multiGetResponseActionFuture;
+	@Mock private MultiGetResponse multiGetResponse;
+	@Mock private MultiGetItemResponse multiGetItemResponse;
+	@Mock private SearchRequestBuilder searchRequestBuilder;
+	@Mock private ActionFuture<SearchResponse> searchResponseActionFuture;
+	@Mock private ActionFuture<MultiSearchResponse> multiSearchResponseActionFuture;
+	@Mock private MultiSearchResponse.Item multiSearchResponseItem;
+	@Mock private SearchScrollRequestBuilder searchScrollRequestBuilder;
 
 	@BeforeEach
+	@SuppressWarnings("deprecation") // we know what we are testing
 	public void setUp() {
 		template = new ElasticsearchTemplate(client);
+		initTemplate(template);
```

Comment:
```suggestion
		initTemplate(new ElasticsearchTemplate(client));
```

with the change in the base class.

Reply:
Fixed